root/etc/plex/plex-install

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+DEBUG=${DEBUG:-}
+
+# If we are debugging, enable trace
+if [ "${DEBUG,,}" = "true" ]; then
+  set -x
+fi
+
+# shellcheck source=../../plex-common.sh
+. /plex-common.sh
+
+CACHED_URL=
+[[ -r /config/install/plexmediaserver.url ]] && CACHED_URL=$(< /config/install/plexmediaserver.url)
+if [ -n "$CACHED_URL" ] && [ -f  /config/install/plexmediaserver.deb ]; then
+  echo "Installing previously downloaded plex version..."
+  installFromRawUrl "$CACHED_URL"
```

Comment:
This is by design. Containers lose all of the changes within them unless they are in a volume or mount. In `autoupdate` images, these will fetch but will not delete the installer, so when the container is restarted it doesn’t need to download it again. Technically, it doesn't NEED to be cached, but it might waste a lot of bandwidth. But, just in case someone wants to actually force an install, I'll add an env var FORCE_UPDATE to ignore the cache.

Reply:
I see what you are getting at.  Though, wouldn't this also install the deb if the container is just restarted?  In that case, it is likely installing the same version that is already installed.  Likely the only way to avoid this is to parse the version out of the deb, compare with the installed version, and skip the install if they are the same.

Reply:
Actually, my bad. I've been using `docker run --rm` for too long - forgot that container images are in fact persisted. Ok, then we don't need to cache the installer at all, and it actually simplifies things a lot.